### DISCUSSION TITLE
Discord Bot !restore command

### DISCUSSION THREAD

**Discussion by HelixEvo** [gameserverapp/Platform#419] [Suggestions]
Area:

Discord Bot

Suggestion Summary:

A command to complement the existing !snapshot (server) (backupname)

New command !restore (server) (backupname) (broadcast message) (shutdown in x)

Pre commands I guess:
Broadcast message of server shutting down in (shutdown in x)
Shutdown server
Restore

Post commands:
Start server back up

**Comment by dagbs** [comment]
I believe starting the server post-restore should be optional. I can see cases where you'd want to restore a server but not start it until later (i.e. an event server that's pre-built and designed to be reused).
